h2o-py/tests/pyunit_utils/utilsPY.py

```diff
@@ -3542,7 +3542,7 @@ def build_save_model_DRF(params, x, train, respName):
 
 
 # generate random dataset, copied from Pasha
-def random_dataset(response_type, verbose=True, NTESTROWS=200):
+def random_dataset(response_type, verbose=True, NTESTROWS=200, missing_fraction=0.0, seed=None):
```

Comment:
I don't understand the purpose of this change.  Looks like a valuable addition, but is this a leftover ?

Reply:
This was written by Pasha originally.  I just copied it over and keep adding to it.  Yes, it was a leftover.